src/functionalTest/groovy/com/autonomousapps/jvm/projects/RewriteDependenciesProject.groovy

```diff
@@ -134,7 +134,7 @@ final class RewriteDependenciesProject extends AbstractProject {
 
   final String expectedBuildFile = '''\
     plugins {
-      id 'java-library'
+      id('java-library')
```

Comment:
You probably want to use `"` instead of `'` here, for Kotlin DSL compatibility.

Reply:
You are correct. The Kotlin DSL that gets generated here would be quoted with `"` instead of `'`. However, this test is specifically for the groovy DSL. If I were to write a test for the Kotlin DSL I would use `"`. I can modify this so that the same syntax works regardless of DSL kind if you would like? That is, I would forego using Scribe to quote what needs quoted and we would just end up with `id("my-plugin")` regardless of DSL.